jac-uk/admin
# #1883 Default to using new QT Platform

## What's included?

This PR is part of #1883 Backwards compatibility

The most important change here is that all exercises should now default to using the new QT platform.

Includes the following:
- [x] Default to use new QT Platform
- [x] Give merit list search box a max width
- [x] Work in Progress: Copy to clipboard and Paste from clipboard

## Who should test?
✅ Product owner
✅ Developers

## How to test?

1. Go to any live exercise which includes QTs in shortlisting.
3. Navigate to Tasks and notice that:
  - The side navigation includes links for the relevant QTs
  - The page includes an instruction to 'Select a task on the left' or similar 
4. If your user role includes the 'Can Approve Exercise' permission then:
  - Check there's a button to 'Use Old QT Platform'
  - Press the 'Use Old QT Platform' button and notice:
    - The button has now swapped to say 'Use New QT Platform'
    - The side navigation now includes a link to 'Qualifying Tests' (the individual links to each test have disappeared)
    - You can view and set up QTs using the old QT Platform
  - Press the 'Use New QT Platform' to check you can switch back to using the new QT platform
    - Individual links in side navigation
    - Instruction on page to select a task

<img width="1225" alt="image" src="https://user-images.githubusercontent.com/8524401/220400903-98da178b-3aeb-4484-89a2-916cd69d6016.png">

<img width="1221" alt="image" src="https://user-images.githubusercontent.com/8524401/220401023-4f70265d-27e3-49eb-b0b2-a10fa3b1481d.png">


## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
